### PR TITLE
Add support for Brain.FM

### PR DESCRIPTION
**Pull request by sstgithub**

### CODE REVIEW

**Comment by sstgithub** [comment]
@berrberr Can you please help here - I don't see why Travis CI failed?

File: code/js/modules/Sitelist.js [hunk 1]
```diff
@@ -67,6 +67,7 @@
       "bbc": { name: "BBC Radio", url: "http://www.bbc.co.uk/radio", controller: "BBCRadioController.js" },
       "beatport": { name: "Beatport", url: "https://www.beatport.com" },
       "blitzr": { name: "Blitzr", url: "http://www.blitzr.com" },
+      "brainfm": { name: "BrainFM", url: "http://brain.fm" },
```
Comment (berrberr) [inline code/js/modules/Sitelist.js]: Need to change this line to: `"brain": { name: "BrainFM", url: "http://brain.fm", controller: "BrainFMController.js" },`

The problem is that you included the TLD in your sitename key, which is incorrect (sitename key matches the domain part only, ignoring TLD). This is also why the tests were failing :)

File: code/js/modules/Sitelist.js [hunk 2]
```diff
@@ -67,6 +67,7 @@
       "bbc": { name: "BBC Radio", url: "http://www.bbc.co.uk/radio", controller: "BBCRadioController.js" },
       "beatport": { name: "Beatport", url: "https://www.beatport.com" },
       "blitzr": { name: "Blitzr", url: "http://www.blitzr.com" },
+      "brainfm": { name: "BrainFM", url: "http://brain.fm", controller: "BrainFMController.js" },
```
Comment (berrberr) [inline code/js/modules/Sitelist.js]: The key needs to be `brain` not `brainfm` ;)

**Comment by berrberr** [comment]
Thanks :)

**Comment by sstgithub** [comment]
And thanks to you for the feedback and for making this extension :)

**Comment by charliesneath** [comment]
Not able to get this integration to work, any ideas why?

**Pull request merged**